tokio-threadpool/src/worker/entry.rs

```diff
@@ -29,10 +29,10 @@ pub(crate) struct WorkerEntry {
     next_sleeper: UnsafeCell<usize>,
 
     // Worker half of deque
-    worker: deque::Worker<Arc<Task>>,
+    pub worker: Worker<Arc<Task>>,
```

Comment:
I'm assuming this will also be `pub(crate)` since the struct is `pub(crate)` 👍

Reply:
That is true. We'll clean up `WorkerEntry` sometime in the future as it's currently in a bit of a mess (because e.g. some fields are owned and some are shared with no clear separation).